### DISCUSSION TITLE
How to disable list style

### DISCUSSION THREAD

**Discussion by mikinovation** [tinymce/tinymce#9628] [General]
# version

```
@tinymce/tinymce-vue: v5.1.1
```

# What I want

I want to turn off list style format.

For example, 
```
- test1
```

When I write like this, and type "Enter", it will change to like this.

```
● test1
●
```

However, I want the plain list should be remained.

```
- test1
- test2
```

Is it possible to do it?

Even if I tried to remove "lists" and "advlist" plugins, it won't work

**Comment by TheSpyder** [comment]
This is controlled by `text_patterns`.
https://www.tiny.cloud/docs/tinymce/latest/content-behavior-options/#text_patterns

**Reply by mikinovation** [reply]
Thank you! I resolved it
